package.json

```diff
@@ -35,6 +35,10 @@
     "@babel/plugin-transform-react-constant-elements": "7.9.0",
     "@babel/plugin-transform-react-jsx": "7.9.4",
     "@babel/preset-env": "7.9.0",
+    "@storybook/addon-a11y": "5.3.17",
+    "@storybook/addon-knobs": "5.3.17",
+    "@storybook/addon-viewport": "5.3.17",
+    "@storybook/preact": "5.3.17",
```

Comment:
These dependencies add up to nearly 200M, and they more than double the size of the root `node_modules` directory. This will result in unnecessary downloads during local development, and significantly slow down our Travis builds.

```sh
rsimha@rsimha-macbookpro2 (wassgha-storybook):~/src/amphtml$ npx cost-of-modules --no-install --include-dev

┌─────────────────────────────────────────────────┬───────────────┬─────────┐
│ name                                            │ children      │ size    │
├─────────────────────────────────────────────────┼───────────────┼─────────┤
│ @storybook/preact                               │ 788           │ 101.54M │
├─────────────────────────────────────────────────┼───────────────┼─────────┤
│ @storybook/addon-a11y                           │ 219           │ 47.45M  │
├─────────────────────────────────────────────────┼───────────────┼─────────┤
│ @storybook/addon-knobs                          │ 74            │ 24.54M  │
├─────────────────────────────────────────────────┼───────────────┼─────────┤
│ @storybook/addon-viewport                       │ 11            │ 1.80M   │
├─────────────────────────────────────────────────┼───────────────┼─────────┤
│ Total: 178 modules                              │ 1829 children │ 361.97M │
└─────────────────────────────────────────────────┴───────────────┴─────────┘
rsimha@rsimha-macbookpro2 (wassgha-storybook):~/src/amphtml$ 
```

Instead of putting them in the root `package.json`, can you create a directory level `package.json` in a new folder `build-system/tasks/storybook/`, and do a lazy install of packages used in that directory when the `gulp` tasks for storybook are invoked?

Here's an example:

https://github.com/ampproject/amphtml/blob/e015e817cb7d7159cc11faa41ef8216ab93cd9b9/build-system/tasks/e2e/index.js#L63-L65

https://github.com/ampproject/amphtml/blob/e015e817cb7d7159cc11faa41ef8216ab93cd9b9/build-system/common/utils.js#L133-L146